src/components/table/Table.vue

```diff
@@ -719,6 +731,12 @@ export default {
                     vnode.componentInstance.$data &&
                     vnode.componentInstance.$data._isTableColumn)
                 .map((vnode) => vnode.componentInstance)
+        },
+        canDragRow() {
```

Comment:
Those computed (`canDragRow` and `canDragColumn`) are now used to avoid being able to drop a column into a row, when dragging.